diff --git a/node_modules/wait-on/lib/wait-on.js b/node_modules/wait-on/lib/wait-on.js
index a31681d..d736f9e 100644
--- a/node_modules/wait-on/lib/wait-on.js
+++ b/node_modules/wait-on/lib/wait-on.js
@@ -8,7 +8,7 @@ const net = require('net');
 const util = require('util');
 const axiosPkg = require('axios').default;
 const axiosHttpAdapter = require('axios/lib/adapters/http');
-const { isBoolean, isEmpty, negate, noop, once, partial, pick, zip } = require('lodash/fp');
+const { isBoolean, isEmpty, negate, noop, once, partial, pick, zip } = require('lodash');
 const { NEVER, combineLatest, from, merge, throwError, timer } = require('rxjs');
 const { distinctUntilChanged, map, mergeMap, scan, startWith, take, takeWhile } = require('rxjs/operators');
 
